src/www/app/outline_server_repository/index.ts

```diff
@@ -32,7 +32,13 @@ function staticKeysMatch(a: string, b: string): boolean {
   try {
     const l = staticKeyToShadowsocksSessionConfig(a);
     const r = staticKeyToShadowsocksSessionConfig(b);
-    return l.host === r.host && l.port === r.port && l.password === r.password && l.method === r.method;
+    return (
+      l.host === r.host &&
+      l.port === r.port &&
+      l.password === r.password &&
+      l.method === r.method &&
+      l.prefix == r.prefix
```

Comment:
People are complaining that the app is saying keys with the prefix `prefix=%16%03%01%40%00%01` are invalid. I think there's something wrong with the serialization. Maybe that's not valid utf8/unicode? We need some unit testing.

Reply:
Note that it works with the prod app (I guess we just ignore the prefix), but not the apk you provided. I can give you a key in private.

Reply:
I've updated the instructions to help people avoid writing invalid prefixes.